site/content/blog/2020-03-09-zap-ssrf-setup/index.md

```diff
@@ -45,7 +45,7 @@ If you want to have a valid HTTPS certificate you can run some reverse proxy on
 ssh -N -R 9999:localhost:38193 myUser@something.mydomain.org
 ```
 
-__Troubleshot__:
+### Troubleshoot:
 - open the ports on the firewall
 - make shure you allow port forwarding on the server ssh config `/etc/ssh/sshd_config`:
```

Comment:
`sure`, just noticed while reading in the site.

Reply:
Sorry, I did a new PR

Reply:
No worries.